database/seed.js

```diff
@@ -6,19 +6,19 @@ const seedData = () => {
   for (let i = 0; i < 15; i += 1) {
     // If no song at that index, use the last song (uses faker data to fill in details)
     const randomSong = {
-      songId: Math.floor(Math.random() * 999),
-      length: Math.floor(Math.random() * (500 - 100) + 100),
+      songId: Math.floor(Math.random() * 2000),
+      length: 183,
       timestamp: 0,
       isliked: Math.floor(Math.random() * 2),
-      songfile: faker.internet.url(),
+      songFile: 'https://sound-clout.s3-us-west-1.amazonaws.com/Fleas.mp3',
```

Comment:
Just to confirm: All of your database entries are playing the same song?

Reply:
Only the last few.  The first 12 play different ones, I just rounded it out to 15 by repeating one for the last few

Reply:
(I was thinking I would create a nextUp pop up, and wanted to have a decent number to scroll through)